### ISSUE TITLE
Separate receiveTimeouts for commands and events

### ISSUE THREAD

**Issue by ermanito** [RBMHTechnology/eventuate#264]
There can be use cases in which an actor shall only be kept alive as long as there are incoming commands. For it it would be useful to have something similar like the BehaviourContext where a developer can define differing behaviours for commands and events separately.

Currently an actors' receiveTimeout can be configured globally causing an actor kept alive on event as well.

@krasserm What do you think?

**Comment by krasserm** [comment]
@ermanito this sounds like a useful addition. Do you plan making a contribution?

**Comment by ermanito** [comment]
@krasserm Yes, we would like to contribute this enhancement. What's the best way forward here? Shall we create a small draft how we would implement this so you can comment on this first before implementation? 

Maybe you already have an idea? At a first glimpse i had the idea to do something similar like the "BehaviourContext" or even re-use/extend the "BehaviourContext". What do you think?

**Comment by krasserm** [comment]
@ermanito Glad to hear you're planning a contribution. I don't have a detailed implementation in mind yet but modifying `BehaviorContext` (e.g. adding a `setReceiveTimeout` method) sounds like a good idea. Let's discuss that more detailed in a first pull request. Looking forward to review it 😃
